keyboards/kinesis/kint36/kint36.c

```diff
@@ -0,0 +1,62 @@
+#include "kint36.h"
+#include "chprintf.h"
+
+void matrix_init_kb(void) {
+    led_init_ports();
+    matrix_init_user();
+}
+
+#define ledTurnOff palSetLine
+#define ledTurnOn palClearLine
```

Comment:
please convert to GPIO control functions: 
https://docs.qmk.fm/#/internals_gpio_control

Reply:
Done